docs/static/releasenotes.asciidoc

```diff
@@ -48,6 +49,193 @@ This section summarizes the changes in the following releases:
+
+* Update bundled JDK to 11.0.11+9 https://github.com/elastic/logstash/pull/12881[#12881]
+* Update JRuby to 9.2.19.0 https://github.com/elastic/logstash/pull/12989[#12989]
+
+[[plugins-7-14-0]]
+==== Plugins
+
+*Csv Filter - 3.1.1*
+
+* Refactor: unified ECS target + validate field reference https://github.com/logstash-plugins/logstash-filter-csv/pull/86[#86]
+
+* Add ECS support https://github.com/logstash-plugins/logstash-filter-csv/pull/85[#85]
+
+* [DOC] Fixed formatting to improve readability https://github.com/logstash-plugins/logstash-filter-csv/pull/84[#84]
+
+*Fingerprint Filter - 3.3.2*
+
+* [DOC] Clarify behavior when key is set https://github.com/logstash-plugins/logstash-filter-fingerprint/pull/65[#65]. 
+
+* Force encoding to UTF-8 when concatenating sources to generate fingerprint https://github.com/logstash-plugins/logstash-filter-fingerprint/pull/64[#64]
+
+* Add ECS compatibility https://github.com/logstash-plugins/logstash-filter-fingerprint/pull/62[#62]
+
+*Geoip Filter - 7.2.2*
+
+* [DOC] Add documentation for database auto-update behavior and database metrics https://github.com/logstash-plugins/logstash-filter-geoip/pull/187[#187]
+
+* Republish the gem due to missing jars in 7.2.0 https://github.com/logstash-plugins/logstash-filter-geoip/pull/186[#186]
+
+*Json Filter - 3.2.0*
+
+* Feat: check target is set in ECS mode https://github.com/logstash-plugins/logstash-filter-json/pull/49[#49]
+* Refactor: logging improvements to print event details in debug mode
+
+*Useragent Filter - 3.3.1*
+
+* Fix: invalid 3.3.0 release which did not package correctly https://github.com/logstash-plugins/logstash-filter-useragent/pull/71[#71]
+
+* Feat: support ECS mode when setting UA fields https://github.com/logstash-plugins/logstash-filter-useragent/pull/68[#68]
+ 
+* Fix: capture os major version + update UA regexes https://github.com/logstash-plugins/logstash-filter-useragent/pull/69[#69]
+
+* Plugin no longer sets the `[build]` UA version field which is not implemented and was always `""`.
+* Fix: `target => [field]` configuration, which wasn't working previously
+
+*Azure_event_hubs Input - 1.3.0*
+
+* Add EventHub `user properties` in `@metadata` object https://github.com/logstash-plugins/logstash-input-azure_event_hubs/pull/66[#66]
+
+*Beats Input - 6.1.6*
+
+* [DOC] Applied more attributes to manage plugin name in doc content, and implemented conditional text processing. https://github.com/logstash-plugins/logstash-input-http/pull/423[#423]
+
+*File Input - 4.3.1*
+
+* Add extra safety to `chown` call in `atomic_write`, avoiding plugin crashes and falling back to a 
+`non_atomic_write` in the event of failure https://github.com/logstash-plugins/logstash-input-file/pull/295[#295]
+* Refactor: unify event updates to happen in one place https://github.com/logstash-plugins/logstash-input-file/pull/297[#297]
+* Test: Actually retry tests on `RSpec::Expectations::ExpectationNotMetError` and retry instead of relying on timeout
```

Comment:
Not relevant to Logstash release

Reply:
We're more permissive with what we leave in for plugin changelogs.  The ECS PR doesn't explain why the plugin is at 4.3.1.